android/conf/local.conf.js

```diff
@@ -0,0 +1,19 @@
+USERNAME =  process.env.BROWSERSTACK_USERNAME || 'BROWSERSTACK_USERNAME',
+ACCESS_KEY = process.env.BROWSERSTACK_ACCESS_KEY || 'BROWSERSTACK_ACCESS_KEY',
+  
+exports.capabilities = {
+    "platformName" : "android",
+    "appium:deviceName" : "Samsung Galaxy S22 Ultra",
+    "browserName" : "chrome",
```

Comment:
??

Reply:
- For nodejs/mocha we had two options of solutions that is built for appium i.e webdriverio and wd.
- We already have webdriverio sample repo and wd is deprecated.
- Alternative is to use selenium driver only and for that we have to use the browser cap (limitation). This same set of caps are used in node sdk too.